i18n/en/docusaurus-plugin-content-docs/current/installation/linux.mdx

```diff
@@ -0,0 +1,200 @@
+---
+sidebar_position: 2
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Linux (Debian, Ubuntu, Raspberry Pi)
+
+:::caution
```

Comment:
I feel like this would be a good warning to also have in the DE version, seeing as this page specifically leans on APT for installation.

Unless we rephrase this page entirely for all Linux installations?

Reply:
Yes, good point. I think it's sufficient to do a `note` (instead of caution) here. I've created an issue for this and will adjust both.